YandexTests/SignIn.cs

```diff
@@ -0,0 +1,49 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using Yandex;
+using System.Threading;
+
+namespace YandexTests
+{
+    [TestClass]
+    public class SignIn
+    {
+        readonly ConfigData _config;
+
+        public SignIn()
+        {
+            _config = new ConfigData();
+        }
+
+        [TestMethod]
+        public void SignInPositive()
+        {
+            var user = "user1";
+
+            IWebDriver driver = _config.Driver;
+            
+            /* Set driver window position and size, if you need. */
+            //_config.SetDriverWindow();
+
+            // Go to Start page
+            StartPage startPage = StartPage.GoToStartPage(_config);
+
+            // Go to Username form to get to Mail page
+            PassportUsernamePage passportUsernamePage = startPage.GoToPassportUsernamePage();            
+
+            // Enter username
+            PassportPasswordPage passportPasswortPage = passportUsernamePage.EnterUsername(_config.Users[user].Username);
+
+            // Enter password and sign in
+            MailPage mailPage = passportPasswortPage.EnterPassword(_config.Users[user].Password);
+            
+            // Verify that user is signed in
+            Assert.IsTrue(driver.FindElement(By.XPath(MailPage.InboxButtonXPath)).Displayed);
```

Comment:
Don't forget to add the second parameter here. For example, "Element Inbox button is not displayed". This parameter will provide more information about reason of test failure at this line in case if it happens.